src/pages/index.js

```diff
@@ -156,23 +157,22 @@ function handleAvatarChange(data) {
       userInfo.setUserInfo(data)
     })
     .then(avatarPopup.close)
-    .finally(avatarPopup.renderLoading(false))
+    .finally(() => avatarPopup.renderLoading(false))
     .catch((err) => {
       console.log(console.error);
     });
 
 }
 
 function handleAddCardSubmit(data) {
+  addCardPopup.renderLoading(true);
   api.uploadCard(data)
     .then((res) => {
-      const card = createCard(res);
-      addCardPopup.renderLoading(true);
+      const card = createCard(res.data);
       cardSection.addItem(card);
     })
-    .then(addCardPopup.close)
-    .finally(() => {
-      addCardPopup.renderLoading(false)})
+    .then(addCardPopup.close())
```

Comment:
`.then(addCardPopup.close())` should be `.then(addCardPopup.close)` 


In blocks `then`, `catch`, `finally` you need to pass only the **reference** to the function (not a function call) otherwise the function is called immediately without waiting for the response from the server. You can pass an **unnamed** arrow function as well.